src/components/cylc/Header.vue

```diff
@@ -55,8 +56,28 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           :default="owner"
           :items="Array.from(owners)"
           v-model="owner"
-          @keyup.enter="owners.add(owner)"
-        />
+          @keyup.enter="addOwner(owner)"
+        >
+          <template v-slot:item="{ item, props }">
+            <!-- HTML that describe how select should render items when the select is open -->
+            <div class="d-flex">
+              <v-list-item
+                class="flex-grow-1"
+                :title="item.title"
+                v-bind="props"
+                >
+                <template v-slot:append v-if="item.title !== ownerOnLoad">
+                    <v-icon
+                      @click.stop="removeOwner(item.title)"
+                      color="pink-accent-4"
+                      :icon="mdiClose"
+                    />
+                </template>
+              </v-list-item>
+            </div>
```

Comment:
The `<div>` here is superfluous
```suggestion
            <v-list-item
              :title="item.title"
              v-bind="props"
            >
              <template v-slot:append v-if="item.title !== ownerOnLoad">
                <v-icon
                  @click.stop="removeOwner(item.title)"
                  color="pink-accent-4"
                  :icon="mdiClose"
                />
              </template>
            </v-list-item>
```